src/App.js

```diff
@@ -19,20 +19,17 @@ function App() {
   const searchHandler = async () => {
     setIsLoading(true);
     try {
-      await fetch(
+      const response = await fetch(
         `${api.base}weather?q=${search}&units=metric&APPID=${api.key}`
-      )
-        .then((res) => res.json())
-
-        .then((result) => {
-          setWeather(result);
-          setIsLoading(false);
-        });
+      );
+      const data = await response.json();
+      setWeather(data);
+      setIsLoading(false);
```

Comment:
you can remove loader after you fetch background below

Reply:
You want me to not have loader?

Reply:
sorry my bad, setIsLoading(false) is "removing" loader, so what i ment is to move setIsLoading(false); after you fetch background down below. It makes more sence to show loader while you are loading weather api responce and background, not just weather  api responce